it/mail/chrome/messenger/preferences/downloads.dtd

```diff
@@ -1,24 +1,24 @@
-<!ENTITY askOnSave.label              "Attachments Folder">
-<!ENTITY alwaysAsk.label              "Ask me where to save every file">
+<!ENTITY askOnSave.label              "Cartella allegati">
+<!ENTITY alwaysAsk.label              "Chiedimi dove salvare ogni file">
 <!ENTITY alwaysAsk.accesskey          "A">
-<!ENTITY neverAsk.label               "Save all attachments to this folder:">
+<!ENTITY neverAsk.label               "Salva tutti gli allegati in questa cartella:">
 <!ENTITY neverAsk.accesskey           "S">
-<!ENTITY desktop.label                "Desktop">
+<!ENTITY desktop.label                "Scrivania">
 <!ENTITY downloads.label              "Download">
 <!ENTITY chooseFolderWin.label        "Sfoglia…">
 <!ENTITY chooseFolderWin.accesskey    "S">
-<!ENTITY chooseFolderMac.label        "Choose...">
+<!ENTITY chooseFolderMac.label        "Scegli...">
 <!ENTITY chooseFolderMac.accesskey    "C">
 
 
-<!ENTITY fileTypes.label              "Download Actions">
-<!ENTITY configureActions.label       "View &amp; Edit Actions...">
+<!ENTITY fileTypes.label              "Azioni download">
+<!ENTITY configureActions.label       "Visualizza e Modifica azioni...">
```

Comment:
CORRECTIONS NEEDED. The correct entry should be as follows: 

+<!ENTITY configureActions.label       "Visualizza e modifica Azioni...">